src/fixtures/CustomMatchers.ts

```diff
@@ -0,0 +1,52 @@
+import { expect as baseExpect, Locator } from '@playwright/test';
```

Comment:
Could you import Locator as type?

```suggestion
import type { Locator } from '@playwright/test';
```

Reply:
Done